scripts/build.py

```diff
@@ -144,8 +144,13 @@
     dest='minify_third_party_libs_only')
 
 
-def generate_app_yaml():
-    """Generate app.yaml from app_dev.yaml."""
+def generate_app_yaml(is_prod_mode=False):
+    """Generate app.yaml from app_dev.yaml.
```

Comment:
This needs more explanation. Can you please explain here how the resulting app.yaml differs in the prod_mode and non-prod_mode case (and, more importantly, the rationale behind that behaviour)?